docs/source/user/api_change.rst

```diff
@@ -10,13 +10,69 @@ The line number corresponds to the resulting line number after all changes are i
 Thus, be sure to implement each in order so that subsequent line numbers are correct.
 
 
-OpenFAST v3.2.0 to OpenFAST `dev`
+OpenFAST v3.3.0 to OpenFAST `dev`
 ----------------------------------
 
 None
 
 
 
+OpenFAST v3.2.0 to OpenFAST v3.3.0
+----------------------------------
+
+============================================= ==== ================= ======================================================================================================================================================================================================
+Added in OpenFAST dev 
```

Comment:
Change "OpenFAST dev" to "OpenFAST v3.3.0"; likewise below.

Reply:
Thanks, fixed